### PR TITLE
feat: add clusterName as a required field

### PR DESCRIPTION
**Pull request by nirbenator**

### CODE REVIEW

File: charts/k8s-watcher/templates/configmap.yaml [hunk 1]
```diff
@@ -7,4 +7,5 @@ metadata:
   {{- end }}
 data:
   komodor-k8s-watcher.yaml: |
-{{- include "watcher.values" . | nindent 4 }}
\ No newline at end of file
+{{- include "watcher.values" . | nindent 4 }}
+    clusterName: {{ .Values.clusterName | required "Clustername is rquired" }}
```
Comment (Moran-k) [inline charts/k8s-watcher/templates/configmap.yaml:11]: Add to readme

Comment (undera) [inline charts/k8s-watcher/templates/configmap.yaml:11]: Our `clusterName` lives under `watcher` AFAIK. Is there a mistake to use `.Values.clusterName`?

Comment (undera) [reply charts/k8s-watcher/templates/configmap.yaml:11]: Also a misprint there in `Clustername is rquired` and also `Clustername`!=`clusterName`

Comment (nirbenator) [reply charts/k8s-watcher/templates/configmap.yaml:11]: whops for both , good catch :)

**Comment by nirbenator** [comment]
public: watcher.clusterName is now a required value

**Pull request merged**